_posts/2018-07-10-Play Framework-in-Scala–Action-Composition.md

```diff
@@ -0,0 +1,256 @@
+I write this post as a way to consolidate my knowledge on Actions.
+Most of what I write is explained on the Play Framework <a href="#play_docs"> docs</a>, although I believe you can benefit from my explanation. The last chapter, <strong><a href="#chaining_actionbuilders">Chaining ActionBuilders</a></strong>, is the most interesting. This topic is under-explained on the docs, making it the most valuable.
+
+<h2> Overview </h2>
+
+In an mvc framework, the Controllers are one of the 3 layers of the mvc. They are a bridge between the incoming requests and the business logic.
```

Comment:
I'm not sure if it should be "an MVC" or "a MVC", but I think it's the latter. Also remove "of the MVC" from the end of the sentence as it's implicit.
Please capitalize MVC as well.